terraform/shared/shared.tf.json.template.py

```diff
@@ -378,9 +382,43 @@ def paren(s: str) -> str:
                 }
             }
         },
-        **(
-            {
-                'aws_cloudformation_stack': {
+        'aws_cloudformation_stack': {
```

Comment:
This should go after the chatbot resource. Without a good reason to prepend, if in doubt, append new code.